data/SamuelIgwesi.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "SAMUEL IGWESI",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Web Developer & Tech Enthusiast",
+  "avatar": "https://github.com/SamuelIgwesi.png",
+  "tags": ["javascript", "nodeJS"],
+   "socials": [
+    { "platform": "twitter", "url": "https://twitter.com/prime_developer" },
+    { "platform": "github", "url": "https://github.com/Samuel_Igwesi" },
```

The GitHub link seems to be wrong here.
```suggestion
    { "platform": "github", "url": "https://github.com/SamuelIgwesi" },
```

---

```suggestion
    { "platform": "github", "url": "https://github.com/SamuelIgwesi" },
```